scoap3/articles/api/views.py

```diff
@@ -40,6 +42,9 @@ def create(self, request, *args, **kwargs):
         data = request.data
         article_id = data.get("id")
 
+        if not request.user.has_perm("articles.add_article"):
```

Comment:
lets do something like this https://www.django-rest-framework.org/tutorial/4-authentication-and-permissions/#object-level-permissions (check `SAFE_METHODS` allow bypassing for GET, HEAD, etc - safe actions)

we can check the `request.user.groups` and `self.action` to determine if allowed or not